tests/Integration/Http/ResourceTest.php

```diff
@@ -51,7 +50,8 @@ public function testResourcesMayBeConvertedToJson()
         });
 
         $response = $this->withoutExceptionHandling()->get(
-            '/', ['Accept' => 'application/json']
+            '/',
+            ['Accept' => 'application/json']
```

Comment:
please revert all these unrelated changes

Reply:
My bad I used my VSCode formatted, should be good in the last commit #38332